### DISCUSSION TITLE
Is it possible to set version_format globally?

### DISCUSSION THREAD

**Discussion by theherk** [starship/starship#4297] [Q&A] [answered]
To me, there is no mystery that x.y.z is a version. I don't like to add "v" prefixes unless there is ambiguity that warrants it. So I set `version_format = "${raw}"` almost everywhere. Is it possible to set this once?

**Comment by davidkna** [comment, marked as answer]
It's not possible to set it all at once, but you could try making use of the `stasrhip print-config` command.

**Reply by theherk** [reply]
Okay, thank you. That at least allows me to write one command to update them all.
